src/components/Routes/AppRoutes.tsx

```diff
@@ -0,0 +1,41 @@
+import { Routes, Route, Navigate } from 'react-router-dom';
+
+import Dashboard from 'pages/Dashboard/Dashboard';
+import Subject from 'pages/Subject/Subject';
+import AddResult from 'pages/AddResult/AddResult';
+import { isAuthenticated } from 'utils/auth';
+import Login from 'pages/Login/login';
+import Signup from 'pages/Signup/signup';
+import ProtectedRoute from 'components/Routes/ProtectedRoute';
+
+const AppRoutes = () => {
+  return (
+    <Routes>
+      <Route
+        path="/"
+        element={
+          isAuthenticated() ? <Navigate to="/dashboard" replace /> : <Login />
```

Comment:
do we need to call each time this function? can't we store this value in a variable ?

Reply:
@atharateeq-mergestack ?